homeassistant/components/emulated_hue/hue_api.py

```diff
@@ -556,16 +561,25 @@ async def put(self, request, username, entity_number):
                     create_hue_success_response(entity_number, val, parsed[key])
                 )
 
-        # Echo fetches the state immediately after the PUT method returns.
-        # Waiting for a short time allows the changes to propagate.
-        await asyncio.sleep(0.25)
-
         return self.json(json_response)
 
 
 def get_entity_state(config, entity):
     """Retrieve and convert state and brightness values for an entity."""
-    cached_state = config.cached_states.get(entity.entity_id, None)
+    cached_state_entry = config.cached_states.get(entity.entity_id, None)
+    cached_state = None
+
+    # Check if we have a cached entry, and if so if it hasn't expired.
+    if cached_state_entry is not None:
+        if (
+            cached_state_entry[1] is None
+            or time.time() - cached_state_entry[1] < STATE_CACHED_TIMEOUT
+        ):
+            cached_state = cached_state_entry[0]
```

Comment:
Please make these constants
```suggestion
    if cached_state_entry is not None:
        if (
            cached_state_entry[1] is None
            or time.time() - cached_state_entry[STATE_CACHE_STATE] < STATE_CACHED_TIMEOUT
        ):
            cached_state = cached_state_entry[STATE_CACHE_TIME]
```